Extensions/Today/ButtonWithSublabel.swift

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+class ButtonWithSublabel: UIButton {
+    lazy var subtitleLabel = UILabel()
+
+     lazy var label = UILabel()
+
+     required init?(coder aDecoder: NSCoder) {
+         fatalError("init(coder:) has not been implemented")
+     }
+
+     convenience init() {
+         self.init(frame: .zero)
+     }
+
+     override init(frame: CGRect) {
+         super.init(frame: frame)
+         performLayout()
+     }
+
+     fileprivate func performLayout() {
+         let titleLabel = self.label
+         self.titleLabel?.removeFromSuperview()
```

Comment:
b) Why do we need to removeFromSuperView() and add it as subview?

Reply:
this was from the old code and I didn't need to edit this, I will remove any redundant or unecessary old code as well in this refactor PR 👍🏻

Reply:
@nbhasin2  so removal of title label from the super view is required to remove the default title label of the button and add a custom label that we move around via snap kit. so it's required unless I change the constraints for the default title label of the button using the UIButton title label properties (contentHorizontalAlignment, contentVerticalAlignment and UIEdgeInsets) what do you think? I think it's better to keep it to have more control on the positioning of the title label within the button and also with respect to the subtitle label.

however, the title label that's added to the subview is a new custom added label but the naming is misleading because it has the same name as the default title label of the button.